war/src/main/webapp/images/jenkins-header-logo-new.svg

```diff
@@ -0,0 +1,13 @@
tXyAvEljc36iHxJIxx/h6p0L4OVuFxGFGlFHmRmsTJwbPWj+P54VNZhKj+/rKlkR9tRBTZuMpnHzp4EKeKZLOVc/FiSQdYHag8dmNjwxVA6pg8Ex0gOSkpLinpQuiEgOyP+5q+kFl29kCZV4gt7xe0gRS2Pr1p06ZnRi8OUz76DexD5pc6hJEsiT+BZEsSyFszRR4t3CQbwmPUiB2FueSGazbIpp4sVaN/dYLlioUf2sSostKoYtXKM729fTZiG6aRlmRKGcoTm/pX3BNTSz4JmEmOdp7s+lujuCgAPvoPwRcGk/dK3IKLT/LkTJHnVlS0I3G914EiSZ1Y8hMKgUaoA61eHesIBy8NVm+51693tvnlNWLLC4P28lJHKPjDex98oAlOoHdBjvJLeWyhynYkLsoH0ClpGBlVAgz/tYi9ZevmmFaKjQ2xXvbJx/KyQGx546HZmoGW639AxT/nN0o2AMmfmi8Vl2MzPFc+gI7TUBF7/JIm6icubNvWG202xLpO06xC9reU1i9evC+XFWKrN/h46PhRIPkTHSdDDX6fcSO+0t8Hkr+NT7Ud301tn2yb4Rjo1Op10GUVwMLQnjayisXU03YtzO9WraxE326lilHWKV7NS8jLxWO6dxpJd95BH/4ElC2Lz/Weu9E0ws1IowY72Jm1KNcC8fn0yWfydwlSkDtR4l2IwYH32sUdO1jtXdBlNitBkHQJo2PjWsTmcaTTO3W6u3uNbAsTUoZ/WnC2To6fyfI3ArHVG7h3714GWG6L/tRDju3G2rp2LKBuUCvJZ+cnYsco9ujISMIvVnNL8yQQW7lkI6nFpdLYU5uXLSsy37sMqfXP3WPkg2TESiDlmrjqfjU6NjrlOmzvtmxqKbd3Ihvb7r77bq140NXOc7sFxE7wSJD7p9V9iLLs0TkYx7vHyeR+UVE8HkL7Yp9y6OKQVirCBtc2b7zSbogNsGNLDx482KLW5ct2AbETPKljweC7EAo7bD0pAqYW0stQpAl4o/LYQ0PDjnWDei1VlZX03HAYmxjT07eobfJlu4DYCZ4UpCe0ZrMyUKlN2rGA9DLopCIyDTTnjWA18SRduaCysrKTyi6otrFF3c+X7QJiz/KkICmJY0e8rqihVMStfVRz0SAqU0xEorn2qqpql92r6VhAa7p4sqqA2LM9Fr//NSCJY7FFS7/5xNWY7XQLdcxt5acGoEG6ixjDrTnhqpUrptVqfLaupkuXWpcP23k34WzeVKjqh7CAeks9Jz1q3OmO1eNe2C4qjpfiMmsuYWxiIpZoRzPZiopV7lr/Szt35p0GsoDY7sfo2odxVRw7sqetzdXKW7uqCl3OTFJtpJObFbF16TrGDKNSjpMvZQGxkzwpBBONk2fvefPNJL1ye7hYk4j0E9EsvcjYNSti62ZeHA6P6Oq9XBf/zfLybHMwt6PB4PGGusAJiAea5OlDoZA4iV9tXZ2s8lSpp9gRVgTOBHE8NlLeDe5u3X3kf3/e6u/r7d3qvpjlq1efFQg3lk9QQOwUnhZi0rViEfXHatM2UO37H3xQrfLMthaxo9nDEKd60T8+809vdPd0M/di6WD/wJKx8bGrsJa4VX8BxntRMwT9YY/WFhA7hQcDU9DW6emwC7HbPIvYtOajyA/Ial+dFPnBVqS0tbX1M/aBZBuG+TfJmnjxeIHHTuGplC9f/hbs/AbVpu/sj48NrR7P9bZqqsq5rFwJB+Y0Ae/G052h0M/S7OaJ5gXETuExMJIofAAd0hFmld379tsp9M5NEzc7smpVGhI7wzjs8/vuRACjP8vN7Od/1gJip3gPkW7zJXdT8tleBTdiV1UmRmwqofDbD/PzbxX7fZs6Q10tJ4LBOGmQV69VN68Cj627K5q6JRXLXx08f4FRPiPiBWy0vdmmaemNKrfITxe8HqzGf8Ii6hmzyH+IfqPemPnCzKJAsVO8j1boWkP8Qm3effq0AEKoVZ7YhkgvLtTZFVc4LFKj8/S9dyIU2n+5ITUvroDYaaAiPtcvu5vv8RDVZmKjs3294uOTIQHph2Oqq1dXC9XhgAcNMTOrQZRjgDzbKSB2Gg+s3FzWCuR2uFZ5gc+ethC6T3wc6hLDQ0MOMZ96eetqa9VdYLaRthbSOYB392a1zfXutHMzMwZoXLF8Oc047ZBQZ5CejcHVFy0qFZVVVXEmo5mc6fjYqOhH8qfziH2dIKqC4/QffPChCAZjMYUg5T6GoJ95vUh0XKCyg7VDAdK5A411dXdB77FL12cZ8r1s236T2L79ZvHJGz+J0MONCAe4sB9FUuehoYsWZeZ2MmDGrcPvHxbvH3pfvA3x5Dm8CDYY4hXIqe+w9y+jjQJip/kwN2/eXHzxfH9PKsEwS0HF16+/SiBbA+L+XWPF/lu3rlZUzCJ6002Hca/H4Ws5jNjXEzCbVTWKavtBhBA+hXDBp05+jLDBIYFg+lamLrWNug2pyF7IqrerdZfLdgGx5/Akm2oDzyAT+CNz6Gp14SLuCuQNv+LKK8WamjWiurraSq9cVlYuypF9gCamxfBdnETyVrIbsMhDyhZTTCO24OTklLgIBB5EPkT71z8gTnefRr3L+SXJBH2GeAxSkX9I0iwvDxcQew6PraGhYTOw7L05dPVMFyyC99TU1X62ra3N4THjmQnOcyIFxJ7jDdQF1JnjUNnvZoh/M4qKvnY5yq/lzVzYlY0c9TegRGTXrfiUb4OHzXchNyP1jpnSefT6QaXfQaDOL2LB+ODljNS8/QWKvUBIuDEQqJ4wzW0YrgUY3oLgDSzrc3qPrbgoZjseMuxAil7tCHXsW6DL9fwwBcTO4CNCSpEl5uRkvWkYNTOmuUbMiBqEK1mDWB3VkEggY4C5BKHUylnCerAc+cmLGLpdmRLeD+S0NcU4KmGnYoybhhiDpTVtVmhG249+FzB+P/peQJszSNfRLYqKehDcp+dyp8rKfYrb/H9mnVvdFd6lqwAAAABJRU5ErkJggg=="></image>
```

Comment:
This is an SVG of a base64 encoded PNG

Seems like the least efficient way to store a png

Reply:
Good catch. I changed it to a SVG. I also optimized it through a SVGO frontend